### ISSUE TITLE
View issues found by a single analyzer

### ISSUE THREAD

**Issue by kendrahavens** [dotnet/roslyn#34929]
[Internal Ask]
This request is from an analyzer writer. Add a "find similar issue in file" link in the (Ctrl+.) preview menu. This will open a window that returns all places where the analyzer reports an issue. You can scope it to the file, project, or solution.

**Scenario**
> I wrote an infinite recursion call detection analyzer. Once I author that analyzer I need to have this window to know what cases my analyzer is catching.

**Comment by sharwell** [comment]
I typically use AnalyzerRunner for this (it's a tool in the Roslyn repository which you can use by cloning Roslyn and building from source).

Inside the IDE, I find the best way to find all cases reported by an analyzer is apply the Fix All operation for the "Suppress [id]" feature. Then I can open Git Extensions (or other source control viewer) and see all the changes it made.

**Comment by JosefPihrt** [comment]
You can also download CLI tool [Roslynator.CommandLine](https://www.nuget.org/packages/Roslynator.CommandLine)

and execute command:

```
roslynator analyze/fix PATH_TO_SLN ^
 --analyzer-assemblies PATH_TO_ANALYZER_DLL ^
 --supported-diagnostics DIAGNOSTIC_ID
```

**Comment by CyrusNajmabadi** [comment]
Not planing on doing this in VS.
